src/loader/bgw_launcher.c

```diff
@@ -48,12 +48,36 @@ typedef enum AckResult
 	ACK_SUCCESS,
 } AckResult;
 
+/* See state machine in README.md */
+typedef enum SchedulerState
+{
+	/* Scheduler should be started but has not been allocated or started */
+	ENABLED = 0,
+	/* The scheduler has been allocated a spot in timescaleDB's worker counter */
+	ALLOCATED,
+	/* Scheduler has been started */
+	STARTED,
+
+	/*
+	 * Scheduler is stopped and should not be started automatically. START and
+	 * RESTART messages can re-enable the scheduler.
+	 */
+	DISABLED
+} SchedulerState;
+
 #ifdef TS_DEBUG
 #define BGW_LAUNCHER_RESTART_TIME 0
 #else
 #define BGW_LAUNCHER_RESTART_TIME 60
 #endif
 
+#ifdef TS_DEBUG
+#define BGW_LAUNCHER_POLL_TIME_MS 10L
```

Comment:
Can you add the time suffix to the definitions above too?

Reply:
Done.